players/vlc-rpc.js

```diff
@@ -107,6 +107,11 @@ module.exports =
 		this.command(['volume', value * 256], cb);
 	},
 
+	setSpeed: function(value, cb)
+	{
+		// not implemented
```

Comment:
Please return this message as callback error, otherwise cb is never called. `vlc-rpc.js` code was abandoned at some point in favor of newer communication method used in `vlc.js`. But lets at least add a non-breaking function with proper callback.

Reply:
```javascript
cb = cb || noop;
cb(new Error('VLC speed control is not implemented'));
```